src/util.c

```diff
@@ -149,6 +149,26 @@ int write_zeroes(int fd, size_t l) {
         return 0; /* Return == 0 if we could only write out zeroes */
 }
 
+static bool memeqzero(const void *data, size_t length) {
+        /* Does the buffer consist entirely of NULs?
+         * Copied from https://github.com/rustyrussell/ccan/blob/master/ccan/mem/mem.c#L92.
+         */
+
+        const unsigned char *p = data;
+        size_t len;
```

Comment:
a variable "len" and one "length"? brrmmpf. In particular as "len" is actually used more like an index... maybe call this "i"? or "p"? or something?

Reply:
OK.